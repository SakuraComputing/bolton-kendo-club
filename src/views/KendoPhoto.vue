<template>
    <div class="kendo-photo">
        <h1>Kendo Photographs</h1>
        
        <div class="image">
            <img v-for="img in images" v-bind:src="img" alt="No image">
        </div>
    </div>    
</template>

<script>
    export default {
        data() {
            return {
                images:['https://i0.wp.com/kenshi247.net/wp-content/uploads/2016/12/2016-kyoto1.jpg?w=800&ssl=1','http://www.jsnw.org.uk/photos/storage/cache/images/000/769/033-DSC-0403,medium_large.1463599525.jpg']
            }
        }
    }
</script>

<style>

.kendo-photo {
    margin: 8%;
}
</style>

