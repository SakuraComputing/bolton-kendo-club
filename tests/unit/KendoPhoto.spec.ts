import { shallowMount } from '@vue/test-utils'
import KendoPhoto from '@/views/KendoPhoto.vue'

describe('KendoPhoto.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(KendoPhoto);
    expect(wrapper).toMatchSnapshot()
  })
})
