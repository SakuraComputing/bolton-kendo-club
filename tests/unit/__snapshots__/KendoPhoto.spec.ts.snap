// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KendoPhoto.vue renders correctly 1`] = `
<div class="kendo-photo">
  <h1>Kendo Photographs</h1>
  <div>1. Kyoto Taikai</div>
  <div class="image"><img src="[object Object]" alt="No image"><img src="[object Object]" alt="No image"></div>
  <div>2. Wilf Swindell's sensei founder KNMS</div>
</div>
`;
